### PR TITLE
PHEE-452 Create Template for ph-ee-dpg

### PR DESCRIPTION
**Pull request by Anover000**
[PHEE-452 Create Template for ph-ee-dpg](https://fynarfin.atlassian.net/browse/PHEE-452?atlOrigin=eyJpIjoiNGQyYmY2NDQzYmRjNDdmNjg0ZGU0MjNmODExOTc1ZGIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

@fynmanoj

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -0,0 +1 @@
+conductor.server.host=http://conductor-server.sandbox.fynarfin.io/api/
```
Comment (fynmanoj) [inline src/main/resources/application.properties]: Can we keep this in Yaml Format?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,373 @@
+Mozilla Public License Version 2.0
```
Comment (fynmanoj) [inline LICENSE]: Check the license part once again

**Review by fynmanoj — CHANGES_REQUESTED** [review]

**Review by fynmanoj — CHANGES_REQUESTED** [review]
Please copy and add PR guideline template from integration test repo

**Comment by Anover000** [comment]
> Please copy and add PR guideline template from integration test repo

Done

File: src/main/resources/application.properties (hunk 1 above)
Comment (Anover000) [reply src/main/resources/application.properties]: Done

File: src/main/java/org/mifos/dpgtemplate/conductorsampleworkers/AddNumbersWorker.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    @Override
+    public String getTaskDefName() {
+        return taskDefName;
+    }
+
+    @Override
+    public TaskResult execute(Task task) {
+        TaskResult result = new TaskResult(task);
```
Comment (fynmanoj) [inline src/main/java/org/mifos/dpgtemplate/conductorsampleworkers/AddNumbersWorker.java:22]: Not very sure whether the sample code can created conflicts when added to multiple services and missed to be removed.

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**